crates/virtio-bindings/src/virtio_net.rs

```diff
@@ -583,7 +583,7 @@ fn bindgen_test_layout_virtio_net_config() {
     );
 }
 #[repr(C)]
-#[derive(Debug, Default, Copy, Clone, PartialEq)]
+#[derive(Default, Copy, Clone)]
```

Comment:
I think the discussion in the other PR was in favor of not removing these, but adding manual implementations.

Reply:
Hmm, I thought those issues will come only with a newer version of bindgen and 0.49.0 should be safe. And now I am really not sure what was done earlier when these bindings were generated and merged with 0.49.0. They should have kept a separate patch to let us know :)

Reply:
@slp can you help with some context here? I saw you added the bindings in the old repository.

Reply:
@lauralt I now moved back to kernel v5.0 too and those issues are gone.

What we have now is
kernel: v5.0
bindgen: 0.49.0

Hope this will work fine now and the other thread can conclude about the updates to newer kernel and bindgen.,